src/style.css

```diff
@@ -1 +1,89 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
```

Comment:
You can use the same font-family because that's what it is but with different font-weights